### DISCUSSION TITLE
Commonsense Knowledge - Challenge

### DISCUSSION THREAD

**Discussion by eduardoblanco** [eduardoblanco/csc696H-Spring24#9] [General]
Here are two questions that shouldn't be too hard to answer:

1. Search [ConceptNet](https://conceptnet.io/c/en/car) or any of the many [commonsense resources by AI2](https://mosaickg.apps.allenai.org/) and point out something in the knowledge base that doesn't make sense to you.

1. Can you think of some "commonsense fact" that is more or less likely to be true depending where you are? 

   I will start:
      * _Eduardo is driving the car, so you will see his face better if you take a picture from the left side of the car_.
      * _You won't see live wires near water_. Well, I saw something [like this](https://www.reddit.com/r/WTF/comments/1ugwbh/there_are_live_wires_attached_to_my_shower_head/) with my own eyes recently. Apparently they are very common is some parts of the world.

**Comment by devsawgrad** [comment]
```
Search [ConceptNet](https://conceptnet.io/c/en/car) or any of the many [commonsense resources by AI2]>  
(https://mosaickg.apps.allenai.org/) and point out something in the knowledge base that doesn't make sense to you.
```
THe "parking lot is  AtLocation car " which doesn't make sense. 
![parkinglotatcar](https://github.com/eduardoblanco/csc696H-Spring24/assets/123234921/75310916-f27a-4693-bc55-afc26e5721d2)

Another example : in the ATOMIC2020 dataset, " PersonX activates a shadow over everyone is hindered by PersonX is not a real character" which is confusing. I assume the personX is an actual entity whether virtual or real.

![comet2020_personshadow_hinderedby](https://github.com/eduardoblanco/csc696H-Spring24/assets/123234921/d745d5da-ebf0-4b1a-aeeb-ed478c38dd64)

**Reply by eduardoblanco** [reply]
Nice examples!

**Comment by elliot-j** [comment]
While not necessarily nonsensical ,  I do find it funny that one of the locations a bicycle can be is in the netherlands 

![image](https://github.com/eduardoblanco/csc696H-Spring24/assets/17621302/b2d3d0dd-b843-4859-bb0d-bfce71b95a79)

**Reply by eduardoblanco** [reply]
Yeah. I would say that there are many bikes there, but a random bike is quite unlikely to be there.

**Comment by eduardoblanco** [comment]
I am going to start tagging random students... although I really would like all of you to participate without me tagging anyone.

@jacobquintero and @Abby-OGV Can you share examples for (2)? You can make them up. They do not need to be related to your personal experience. If you need inspiration, browse existing Knowledge graphs until something appears to be only true in some place.

**Comment by jacobquintero** [comment]
Although not incorrect these two examples seem to be US-centric which I found interesting. ( legal drinking age in US is 21 and from a quick google search driving age in most countries is not 16, like it is in the US)
<img width="726" alt="seen as US-centric" src="https://github.com/eduardoblanco/csc696H-Spring24/assets/55100198/51cb8e03-8a0b-480b-aebe-3f2086c2a454">
<img width="1123" alt="Hindered by US-centric" src="https://github.com/eduardoblanco/csc696H-Spring24/assets/55100198/030824c3-f95c-4d13-adde-930a85b79996">

Also, I found this example funny. 

<img width="852" alt="grad-student-funny" src="https://github.com/eduardoblanco/csc696H-Spring24/assets/55100198/34d6b0db-e22d-49c8-a8d5-e54b9299f3cd">

**Reply by eduardoblanco** [reply]
Nice ones!

I know of a country were the legal age is 16 for some alcoholic drinks. Not that anybody would ask for id anyway.

**Comment by Abby-OGV** [comment]
For 1, I found that this does not make sense
<img width="490" alt="Screenshot" src="https://github.com/eduardoblanco/csc696H-Spring24/assets/85028406/3baa2ce9-962f-4bd2-a636-abbbb311803b">

For 2, the final price at checkout is not just a sum of the price of the items purchased (sales tax in the US).

**Reply by eduardoblanco** [reply]
I love the example about prices. At a restaurant you pay much more than what the price is supposed to be between taxes and tip.
